minho/p,y개수.js

```diff
@@ -0,0 +1,10 @@
+function solution(s){
+    var answer = true;
+    var count_p=0;
+    var count_y=0;
+    for(var i=0;i<s.length;i++){
+        if(s[i]=='p'||s[i]=='P')count_p++;
+        if(s[i]=='y'||s[i]=='Y')count_y++;
+    }
```

Comment:
```suggestion
    for(let el of s.toLowerCase()) {
        if (el === 'p') count_p++;
        if (el === 'y') count_y++;
    }
```
for문안에 변수는 해당 변수가 무엇을 의미하는지 나타내주는게 좋고 민호님 코드가 틀린것이 아니라 for (let ... of ...) 라는 문법을 쓰면 문자열이 문자열 길이만큼 반복되어 문자열 하나씩 for문을 돌면서 출력이됩니다. 그리고  toLowerCase() 메서드를 쓰면 문자열을 모두 소문자로 바꾸어준다 라는 것을 알려드리고 싶었습니다!